use path in urls

dulmandakh opened the pull request:
<!-- See  https://django-oauth-toolkit.readthedocs.io/en/latest/contributing.html#pull-requests -->
<!-- If there's already an issue that this PR fixes, add that issue number below after 'Fixes #' -->
Fixes #

## Description of the Change

## Checklist

<!-- Replace '[ ]' with '[x]' to indicate that the checklist item is completed. -->
<!-- You can check the boxes now or later by just clicking on them. -->

- [ ] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [ ] `CHANGELOG.md` updated (only for user relevant changes)
- [ ] author name in `AUTHORS`

n2ygk commented:
Hi @dulmandakh,

Please update the checklist and also explain why this PR is necessary. is replacing `re_path()` with `path()` worth the effort? Thanks.

dulmandakh commented:
> Hi @dulmandakh,
> 
> 
> 
> Please update the checklist and also explain why this PR is necessary. is replacing `re_path()` with `path()` worth the effort? Thanks.

Hi. Updated checklist and added brief explanation.

n2ygk reviewed (approved):
Thanks for this. Always nice to make the code more readable.
